geode-core/src/main/java/org/apache/geode/internal/cache/partitioned/rebalance/model/PartitionedRegionLoadModel.java

```diff
@@ -470,21 +475,23 @@ public Move findBestRemove(Bucket bucket) {
     Move bestMove = null;
 
     for (Member member : bucket.getMembersHosting()) {
-      float newLoad = (member.getTotalLoad() - bucket.getLoad()) / member.getWeight();
-      if (newLoad > mostLoaded && !member.equals(bucket.getPrimary())) {
-        Move move = new Move(null, member, bucket);
-        if (!this.attemptedBucketRemoves.contains(move)) {
-          mostLoaded = newLoad;
-          bestMove = move;
+      if (member.canDelete(bucket, partitionedRegion.getDistributionManager()).willAccept()) {
+        float newLoad = (member.getTotalLoad() - bucket.getLoad()) / member.getWeight();
+        if (newLoad > mostLoaded && !member.equals(bucket.getPrimary())) {
+          Move move = new Move(null, member, bucket);
+          if (!this.attemptedBucketRemoves.contains(move)) {
+            mostLoaded = newLoad;
+            bestMove = move;
+          }
```

Comment:
MemberTest looks great! It would be good to also get `PartitionedRegionLoadModel.createRedundantBucket` under unit test in `PartitionedRegionLoadModelJUnitTest`.

Reply:
I am not familiar with that code, and I don't think now is the right time for that. I am happy to submit a new bug for that.  See GEODE-9584